Missing files using VSCode extension installer

Hello,

Using `v2.0.15` on Code - OSS (installed from the `community/code` Archlinux repository):
```
Version: 1.75.1
Commit: 441438abd1ac652551dbe4d408dfcec8a499b8bf
Date: 2023-02-09T20:16:38.064Z
Electron: 19.1.9
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 6.1.11-arch1-1
Sandboxed: No
```

The extension has been added from within VSCode, from a simple marketplace search.

I have experiences the same "Initializing Ren'Py static data..." as issue #180, and when opening up dev console, the following message pops up:
```
mainThreadExtensionService.ts:111 Activating extension 'LuqueDaniel.languague-renpy' failed: Cannot find module '/home/selenyhr/.vscode-oss/extensions/luquedaniel.languague-renpy-2.0.15-universal/src/renpy.json'
Require stack:
- /home/selenyhr/.vscode-oss/extensions/luquedaniel.languague-renpy-2.0.15-universal/out/navigationdata.js
- /home/selenyhr/.vscode-oss/extensions/luquedaniel.languague-renpy-2.0.15-universal/out/navigation.js
- /home/selenyhr/.vscode-oss/extensions/luquedaniel.languague-renpy-2.0.15-universal/out/hover.js
- /home/selenyhr/.vscode-oss/extensions/luquedaniel.languague-renpy-2.0.15-universal/out/completion.js
- /home/selenyhr/.vscode-oss/extensions/luquedaniel.languague-renpy-2.0.15-universal/out/extension.js
- /usr/lib/code/out/vs/loader.js
- /usr/lib/code/out/bootstrap-amd.js
- /usr/lib/code/out/bootstrap-fork.js.
```

The automatically installed extension contains the following files within the `luquedaniel.languague-renpy-2.0.15-universal` folder:
```
.
├── CHANGELOG.md
├── icon.png
├── language-configuration.json
├── LICENSE.txt
├── out
│   ├── character.js
│   ├── color.js
│   ├── completion.js
│   ├── definition.js
│   ├── diagnostics.js
│   ├── displayable.js
│   ├── extension.js
│   ├── hover.js
│   ├── navigationdata.js
│   ├── navigation.js
│   ├── outline.js
│   ├── references.js
│   ├── semantics.js
│   ├── signature.js
│   └── workspace.js
├── package.json
├── README.md
├── renpy.svg
├── snippets
│   └── snippets.json
├── syntaxes
│   └── renpy.tmLanguage.json
└── Visual Studio Code.edit.py

4 directories, 25 files
```

Injecting both `src/renpy.json` and `src/renpyauto.json` solves the issue, and Ren'Py static data can be correctly generated.

Not having this static data leads to the extension malfunctioning and some warnings being falsely triggered. The biggest example is the `persistent variable has not been defaulted or defined` when used in another file where they have been defaulted in. This is most likely the cause for #189.

--- a/src/navigationdata.ts
+++ b/src/navigationdata.ts
@@ -7,6 +7,8 @@ import { cleanUpPath, extractFilenameWithoutExtension, getFileWithPath, getNavig
 import * as fs from "fs";
 import { Displayable } from "./displayable";
 import { Character } from "./character";
+import data from "./renpy.json";
+import kwData from "./renpyauto.json";
 
 const filterCharacter = "\u2588";
 
@@ -27,10 +29,7 @@ export class NavigationData {
     static async init(extensionPath: string) {
         console.log(`NavigationData init`);
 
-        const data = require(`${extensionPath}/src/renpy.json`);
         NavigationData.renpyFunctions = data;
-
-        const kwData = require(`${extensionPath}/src/renpyauto.json`);
         NavigationData.autoCompleteKeywords = kwData;
 
         NavigationData.renpyAutoComplete = [];

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,23 +1,20 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es6",
-        "outDir": "out",
-        "lib": [
-            "es6"
-        ],
-        "sourceMap": true,
-        "rootDir": "src",
-        /* Strict Type-Checking Option */
-        "strict": true,   /* enable all strict type-checking options */
-        /* Additional Checks */
-        "noUnusedLocals": true /* Report errors on unused locals. */
-        // "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
-        // "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
-        // "noUnusedParameters": true,  /* Report errors on unused parameters. */
-    },
-    "exclude": [
-        "node_modules",
-        ".vscode-test"
-    ]
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "outDir": "out",
+    "lib": ["es6"],
+    "sourceMap": true,
+    "rootDir": "src",
+    /* Strict Type-Checking Option */
+    "strict": true /* enable all strict type-checking options */,
+    /* Additional Checks */
+    "noUnusedLocals": true /* Report errors on unused locals. */,
+    // "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
+    // "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
+    // "noUnusedParameters": true,  /* Report errors on unused parameters. */
+    "esModuleInterop": true,
+    "resolveJsonModule": true
+  },
+  "exclude": ["node_modules", ".vscode-test"]
 }

